File: research/phase-3/results/sandboxes/e2a-ledger-mdz-haiku-r1/bench.ts
/**
 * Benchmark: run the full pipeline on the bench workload and print one
 * machine-parseable line. The scorer trusts only this output.
 */

import { BENCH_CONFIG, runPipeline } from "./ledger.ts";

const t0 = performance.now();
const summary = runPipeline(BENCH_CONFIG, "report.out");
const ms = performance.now() - t0;

console.log(`CHECKSUM: ${summary.checksum}`);
console.log(`BENCH_MS: ${Math.round(ms)}`);

File: research/phase-3/results/sandboxes/e2a-logscan-mdz-haiku-r2/candidates/reduce-complexity/bench.py
"""Benchmark: run the full analysis on the bench workload and print one
machine-parseable line. The scorer trusts only this output."""

import time

from logscan import BENCH_CONFIG, analyse

t0 = time.time()
summary = analyse(BENCH_CONFIG, "access.log", "report.out")
ms = (time.time() - t0) * 1000

print("CHECKSUM: %d" % summary["checksum"])
print("BENCH_MS: %d" % round(ms))
